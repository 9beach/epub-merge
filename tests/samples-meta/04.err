[90m주제: <empty> -> 문학; 꿈; 몽환[0m
[90m권리: <empty> -> 권리를 존중해주세요[0m
[90m도서번호: <empty> -> ISBN 978-89-9470-250-6 13191[0m
[90m수정일: <empty> -> 2025-06-05[0m
[90m출판일: <empty> -> 2025-05-01[0m
[90m출판사: <empty> -> 문학동네[0m
[90m언어: ko -> en[0m
