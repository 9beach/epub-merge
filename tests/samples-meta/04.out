제목: 뱀과물 (Snake And Water)
작가: 배수아 [배, 수아]
작가: Suah Bae [Bae, Suah]
번역: Deborah Smith
출판사: 문학동네
출판일: 2017-12-07
출판일: 2025-05-01
수정일: 2025-06-05
주제: 꿈
주제: 몽환
주제: 문학
설명: <![CDATA[
<p>A dystopian novel about totalitarianism.</p>
<p>Published in 1949.</p>
]]>
언어: en
권리: 권리를 존중해주세요
도서번호: ISBN 978-89-9470-250-6 13191
